3523/0/triangle.rb

```diff
@@ -0,0 +1,49 @@
+# frozen_string_literal: true
+
+# :reek:all
+# rubocop:disable Naming/UncommunicativeMethodParamName
+# rubocop:disable Style/NumericPredicate
+# rubocop:disable Metrics/AbcSize
+# rubocop:disable Metrics/MethodLength
```

Comment:
This file you write by yourself, so you can't disable any errors here, please fix all this errors